### PR TITLE
Ability for custom storyshots testFunctions to utilise "snapshot per story file"

### PR DESCRIPTION
**Pull request by digitalmaster**
## Issue:
Can't use `multiSnapshotWithOptions` with custom test function because `getSnapshotFileName` is not exported.

## What I did
This is a small tweak to the new `multiSnapshotWithOptions` test function. It basically just exports the `getSnapshotFileName` function so it can be used by those of use who have implemented our own custom test function. I also did a minor refactor and moved the `getSnapshotFileName` function to the `utils.js` file. Do let me know if this doesn't make any sense.

## How to test
Go to `storybook\addons\storyshots`
and run `yarn example`

Is this testable with jest or storyshots?
Storyshots

Does this need a new example in the kitchen sink apps?
It should not.

Does this need an update to the documentation?
[Updated Docs] Yes.. We could mention this export in the docs so other users could become aware of it.

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
Fantastic first PR @digitalmaster !

I'd like @igor-dv to have a look before merging 👍

**Review by igor-dv — APPROVED** [review]
Cool. Can you please add an example to the `readme` of how to use it ?

**Comment by digitalmaster** [comment]
@ndelangen Thanks a lot! Glad others also find this useful :)

@igor-dv Added an example.. Let me know if it's unclear or has any bugs. 😬

**Comment by igor-dv** [comment]
@digitalmaster, thanks, looks great !

**Comment by igor-dv** [comment]
@digitalmaster, wanna merge ?

**Comment by digitalmaster** [comment]
Sure.. on to this 3.3 release right? `storybooks:release/3.3`? Just wanna confirm.

**Comment by igor-dv** [comment]
yeah to the 3.3

**Comment by digitalmaster** [comment]
@igor-dv I don't think I can't merge this PR myself without having push access to this repo right? Or am I missing something?

**Comment by igor-dv** [comment]
Ah, so let me push it then 😃

**Comment by digitalmaster** [comment]
Awesome thanks.

**Comment by digitalmaster** [comment]
Do you guys have any idea when another release will be created?

**Pull request merged**
